### ISSUE TITLE
bashew cannot handle path to script

### ISSUE THREAD

**Issue by drmikecrowe** [pforret/bashew#5]
Sorry Peter, another one :)

```
 $  bashew -v script 
# 📜 Script path: /home/mcrowe/.basher/cellar/bin/bashew - folder /home/mcrowe/.basher/cellar/bin 
# 🔗 Linked to: /home/mcrowe/.basher/cellar/packages/pforret/bashew/bashew - folder /home/mcrowe/.basher/cellar/packages/pforret/bashew 
# 🔗 Linked to: /home/mcrowe/.basher/cellar/packages/pforret/bashew/bashew.sh - folder /home/mcrowe/.basher/cellar/packages/pforret/bashew 
# Executing : [/home/mcrowe/.basher/cellar/packages/pforret/bashew/bashew.sh] 
# In folder : [/home/mcrowe/.basher/cellar/packages/pforret/bashew] 
# Detected shell: bash - version 5.0.17(1)-release 
# System    : Distributor ID:   Ubuntu (Linux) Release: 20.10 on x86_64 
# Installer : apt install 
# Expect : 1 single parameter(s): action 
# Found  : action=script 
# No multi param to process 
# all parameters have been processed 
# Cleanup: [.tmp] - delete files older than 1 day(s) 
# Tmpfile: .tmp/2020-12-29.qo0ejz 
# Cleanup: [log] - delete files older than 7 day(s) 
# Logfile: log/bashew.2020-12-29.log 
# Program: bashew 1.10.2 
# Updated: 2020-12-28 07:59 
# Run as : mcrowe@mcrowe-Inspiron-7537 
# Running: on Linux (#33-Ubuntu SMP Mon Nov 23 18:44:54 UTC 2020) 
# Verify : git tput uname  
# Using [scotia_screws.sh] as name 
…  1. first we need the information of the author
Author full name         (Mike Crowe): 
Author email             (drmikecrowe@gmail.com): 
Author (github) username (.): drmikecrowe
…  2. now we need the path and name of this new script/repo
Script name (./scotia_screws.sh): /home/mcrowe/bin/media
…  3. give some description of what the script should do
Script description (This is my script media): Start/Stop media processes
…  Creating script /home/mcrowe/bin/media ...
sed: -e expression #1, char 29: unknown option to `s'
/home/mcrowe/bin/media
# bashew finished after 43 seconds 
```

For reference:
```
 $  sed --version --help
sed (GNU sed) 4.7

 $  sed --help
Usage: sed [OPTION]... {script-only-if-no-other-script} [input-file]...

  -n, --quiet, --silent
                 suppress automatic printing of pattern space
      --debug
                 annotate program execution
  -e script, --expression=script
                 add the script to the commands to be executed
  -f script-file, --file=script-file
                 add the contents of script-file to the commands to be executed
  --follow-symlinks
                 follow symlinks when processing in place
  -i[SUFFIX], --in-place[=SUFFIX]
                 edit files in place (makes backup if SUFFIX supplied)
  -l N, --line-length=N
                 specify the desired line-wrap length for the `l' command
  --posix
                 disable all GNU extensions.
  -E, -r, --regexp-extended
                 use extended regular expressions in the script
                 (for portability use POSIX -E).
  -s, --separate
                 consider files as separate rather than as a single,
                 continuous long stream.
      --sandbox
                 operate in sandbox mode (disable e/r/w commands).
  -u, --unbuffered
                 load minimal amounts of data from the input files and flush
                 the output buffers more often
  -z, --null-data
                 separate lines by NUL characters
      --help     display this help and exit
      --version  output version information and exit

If no -e, --expression, -f, or --file option is given, then the first
non-option argument is taken as the sed script to interpret.  All
remaining arguments are names of input files; if no input files are
specified, then the standard input is read.

GNU sed home page: <https://www.gnu.org/software/sed/>.
General help using GNU software: <https://www.gnu.org/gethelp/>.
E-mail bug reports to: <bug-sed@gnu.org>.
```

**Comment by pforret** [comment]
Ah yes, it's the "/" in your Script description . I have to escape that!

**Comment by pforret** [comment]
Should be fixed in v 1.10.3! I replaced the `sed` chain by one `awk` and it accepts all characters, even: / | ' "
